src/frontend/Komponenter/Behandling/Vilkårpanel/Informasjonsrad.tsx

```diff
@@ -6,23 +6,27 @@ import { FC, ReactNode } from 'react';
 import { mapIkon, VilkårInfoIkon } from './VilkårInformasjonKomponenter';
 
 interface Props {
-    ikon: VilkårInfoIkon;
+    ikon?: VilkårInfoIkon;
     label: string;
-    verdi: ReactNode;
+    verdi?: ReactNode;
     verdiSomString?: boolean;
 }
 
-const InformasjonsradContainer = styled.div`
+const InformasjonsradContainer = styled.div<{ verdiFinnesIkke: boolean }>`
     display: grid;
     grid-template-columns: 21px min(200px, 250px) auto;
     grid-gap: 0.5rem;
+
+    .label {
+        grid-column: 2 ${(props) => props.verdiFinnesIkke && '/ 4'};
+    }
 `;
 
 const Informasjonsrad: FC<Props> = ({ ikon, label, verdi, verdiSomString = true }) => {
     return (
-        <InformasjonsradContainer>
-            {mapIkon(ikon)}
-            <Label size="small" as="h3">
+        <InformasjonsradContainer verdiFinnesIkke={!verdi}>
```

Comment:
```suggestion
        <InformasjonsradContainer harVerdi={verdi}>
```
Dette blir lettere å lese og følger navngivingen vi ellers bruker rundt om kring :) 

Må da sette utropstegnet i sjekken som gjøres ovenfor i `InformasjonsradContainer`